WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteAssemblyContentView.swift

```diff
@@ -0,0 +1,282 @@
+
+import UIKit
+
+// MARK: SiteAssemblyContentView
+
+/// This view is intended for use as the root view of `SiteAssemblyWizardContent`.
+/// It manages the state transitions that occur as a site is assembled via remote service dialogue.
+///
+class SiteAssemblyContentView: UIView {
```

Comment:
Do we really want this class to not be final?

Reply:
Thanks for calling that out. I have pushed an additional commit that marks both `AssembledSiteView` & `SiteAssemblyContentView` as `final`.